CMakeLists.txt

```diff
@@ -59,3 +59,7 @@ endif()
 if (CCCL_ENABLE_EXAMPLES)
   add_subdirectory(examples)
 endif()
+
+if (MSVC)
+  add_cxx_compiler_flag(-use-local-env)
```

Comment:
I don't see this actually making its way down into the lit compile flags: https://github.com/NVIDIA/cccl/actions/runs/6002773902/job/16279832645#step:4:218

Reply:
Yeah, I should not do commits in the evening